core/templates/pages/contributor-dashboard-admin-page/contributor-dashboard-tables/contributor-admin-stats-table.component.spec.ts

```diff
@@ -301,14 +311,48 @@ describe('Contributor stats component', () => {
         'lastContributedInDays',
         'role',
       ]);
+      expect(component.getOverallItemCount()).toEqual('1+');
+    }));
+
+    it('should index last item on page as "overall item count" when "current page\'s cumulative item count" exceeds "overall item count".', fakeAsync(() => {
+      component.statsPageNumber = 1;
+      component.itemsPerPage = 4;
+      fixture.detectChanges();
+      component.allStats = createQuestionReviewerStats(3);
+      expect(component.getIndexOfLastItemOnPage()).toEqual(3);
     }));
 
-    it('should check for upperlimit for pagination', fakeAsync(() => {
-      component.dataSource = [];
+    it('should index last item on page as "current page\'s cumulative item count" when "overall item count" exceeds "current page\'s cumulative item count".', fakeAsync(() => {
```

Comment:
Done. This concern hasn't been raised by python -m scripts.linters.run_lint_checks, since the prettier install, so I assumed this was no longer required. I've changed all the lines to less than 80 chars now.